README.md

```diff
@@ -5,6 +5,7 @@ Exposes an extended Schemable typeclass `SchemableExt` with types inspired by `i
 ## Table of Contents
 
 <!-- AUTO-GENERATED-CONTENT:START (TOC) -->
+
```

Comment:
Not sure why this keeps happening 🤔

Reply:
Maybe it's a Prettier version thing. I'm seeing this happen sometimes as well.